### PR TITLE
Reducing cognitive load and make The list of Methods more readable 

### PR DESCRIPTION
**Pull request by Nir-An**
This PR proposes the following changes to the Laravel Collection documentation:

- Split the methods into common groups: This will make it easier for users to find the methods they need, and to understand how they relate to each other.

- Use simple jargon and English language: The current documentation uses some technical jargon that may be confusing for beginners. This PR replaces this jargon with simpler, more English-friendly language.

- Keep the documentation clean and simple: This PR simplifies the documentation, while still providing all of the necessary information.

### CODE REVIEW

File: collections.md [hunk 1]
```diff
@@ -724,13 +770,13 @@ If you want to stop executing the script after dumping the collection, use the [
 <a name="method-duplicates"></a>
 #### `duplicates()` {.collection-method}
 
-The `duplicates` method retrieves and returns duplicate values from the collection:
+The `duplicates` method retrieves and returns new collection of duplicate values from the collection:
 
-    $collection = collect(['a', 'b', 'a', 'c', 'b']);
+    $collection = collect([5,'a', 'b', 'a', 'c', 'b','5']);
```
Comment (antimech) [inline collections.md:775]:
```suggestion
    $collection = collect([5, 'a', 'b', 'a', 'c', 'b', '5']);
```

File: collections.md [hunk 2]
```diff
@@ -724,13 +770,13 @@ If you want to stop executing the script after dumping the collection, use the [
...
 
-    $collection->duplicates();
+    $duplicated = $collection->duplicates();
 
-    // [2 => 'a', 4 => 'b']
+    // [2 => 'a', 4 => 'b',6 => '5']
```
Comment (antimech) [inline collections.md:779]:
```suggestion
    // [2 => 'a', 4 => 'b', 6 => '5']
```

**Comment by shaedrich** [comment]
Loosely relates to this: https://github.com/laravel/framework/discussions/47044#discussioncomment-7034511

**Comment by Nir-An** [comment]
Hello @taylorotwell 

I hope this message finds you well. The primary objective was to enhance the Laravel Collection documentation by [splitting related methods into distinct groups](https://github.com/Nir-An/docs/blob/d18a7735a9942b5e9218700c18bb312ba446df04/collections.md?plain=1#L78). I believe this approach significantly improves the organization and findability of methods, making them more user-friendly.

Please consider making some changes to make Laravel Collection documentation more
User-friendly mainly for beginners 

Thank you! 

FYI
@jessarcher @nunomaduro

**Comment by shaedrich** [comment]
It's funny how this documentation page reflect the intransparent communication when closing an issue over here.

**Comment by Nir-An** [comment]
> It's almost funny how this documentation page reflect the intransparent communication when closing an issue over here.
@shaedrich 
I don't think so, it is OK to me.
It's perfectly fine to close it And I will make as much effort as necessary to make fixes if they see fit to do so.

In general Please see [code-of-conduct](https://laravel.com/docs/10.x/contributions#code-of-conduct)
special section no.3

**Comment by shaedrich** [comment]
Well, not giving any useful information is neither good nor bad is just lack of necessarily communication.

**Pull request closed without merging**